remappings.txt

```diff
@@ -1,14 +1,15 @@
-test=test/
-@uniswap/=lib/
+@uniswap/=lib/quark/lib/
+codejar/=lib/quark/src/codejar/
 ds-test/=lib/forge-std/lib/ds-test/src/
 erc4626-tests/=lib/openzeppelin-contracts/lib/erc4626-tests/
 forge-std/=lib/forge-std/src/
 openzeppelin-contracts/=lib/openzeppelin-contracts/
 openzeppelin/=lib/openzeppelin-contracts/contracts/
-v3-core/=lib/v3-core/
-v3-periphery/=lib/v3-periphery/contracts/
-codejar/=lib/quark/src/codejar/
+quark-core-scripts/=lib/quark/src/quark-core-scripts/
 quark-core/=lib/quark/src/quark-core/
 quark-factory/=lib/quark/src/quark-factory/
 quark-proxy/=lib/quark/src/quark-proxy/
-src/=src/
\ No newline at end of file
+quark/=lib/quark/
+test/=lib/quark/test/
```

Comment:
I think this should be removed. We should not be referencing test files from the quark submodule.